docs/quickstart.md

```diff
@@ -0,0 +1,226 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+# Preparation
+Please create Hadoop installation and execution environment of MapReduce.
+
+[Apache Hadoop](https://hadoop.apache.org/)
+
+
+## Preparation of input data.
+In this example, JSON data is assumed to be input data.
+With this data as input, we create a table in Yosegi format and read data from the table.
+
+Create the following csv file.In this example, to illustrate writing to and reading from the table, the data is a simple example.
```

Comment:
Create the following CSV file. Following example is for illustrating to write and read simple data from the table.